composer.json

```diff
@@ -43,7 +43,10 @@
         "doctrine/dbal": "< 3"
     },
     "autoload": {
-        "psr-4": { "Doctrine\\Bundle\\FixturesBundle\\": "" }
+        "psr-4": { "Doctrine\\Bundle\\FixturesBundle\\": "src" }
+    },
+    "autoload-dev": {
+        "psr-4": { "Doctrine\\Bundle\\FixturesBundle\\Tests\\": "tests" }
```

Comment:
This is technically a breaking change, because `Tests` is no longer autoloadable by users of this bundle. Looking at the files in there though, there does not seem to be any code that should be.